spec/rspec/core/reporter_spec.rb

```diff
@@ -172,6 +172,27 @@ module RSpec::Core
       end
     end
 
+    describe "#publish" do
+      let(:listener) { double("listener", :custom => nil) }
+      before do
+        reporter.register_listener listener, :custom, :start
+      end
+
+      it 'will send custom events to registered listeners' do
+        expect(listener).to receive(:custom).with(RSpec::Core::Notifications::NullNotification)
+        reporter.publish :custom
+      end
+      it 'will ignore RSpec standard events' do
```

Comment:
Any reason you didn't separate these specs by blank lines?   I think it looks messy to have them crammed together...